lib/Controller/SettingsController.php

```diff
@@ -181,29 +180,36 @@ public function setUser()
 	*/
 	public function getIceServers()
 	{
-		$secret = $this->getAppValue('iceSecret');
+		$secret = $this->getAppValue('iceSecret', '');
```

Comment:
In templates/settings/admin.php you remove the iceSecret field, so how should this work?